wgpu-hal/src/dx12/adapter.rs

```diff
@@ -263,6 +299,40 @@ impl super::Adapter {
             }
         };
 
+        let mut all_shader_models = [
```

Comment:
This isn't immediately clear why SM5.1 didn't need to be in this list. Might be worth a comment, or a rename of the variable to not be _all_ :)

Reply:
Renamed it to `shader_models_after_5_1`.